packages/@aws-cdk/aws-route53-patterns/lib/website-redirect.ts

```diff
@@ -11,23 +11,34 @@ import * as crypto from 'crypto';
  */
 export interface HttpsRedirectProps {
   /**
-   * HostedZone of the domain
+   * Hosted zone of the domain. Container for records which contain information
+   * about how you want to route traffic for a specific domain (example.com)
+   * and it's subdomains (acme.example.com, zenith.example.com). 
+   * 
+   * A hosted zone and the corresponding domain will have the same name.
```

Comment:
You're describing what a hosted zone generally is, but not how this zone is being used. Is this the zone for the "source" domains, or the zone for the targetDomain? Does it have to be both? (Can I redirect example.co.uk to example.com)?